node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+
+const fileName = process.argv.slice(2);
```

Comment:
This is an array, so there's more than one file name. Maybe the variable should be named with a plural to make it more clear?